client/landing/stepper/declarative-flow/internals/steps-repository/playground/components/playground-error/style.scss

```diff
@@ -0,0 +1,33 @@
+.playground-error {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  background-color: #f6f7f7;
+
+  &__content {
+    max-width: 500px;
+    padding: 2rem;
+    text-align: center;
+    background-color: white;
```

Comment:
The builder is saying that "white is an unexpected color name". We should check if we must use a global variable here.

Reply:
@zaerl Can you take a look again? Screenshot in PR description.

Reply:
👍